specs/core/0_beacon-chain.md

```diff
@@ -1088,28 +1075,75 @@ def get_domain(fork: Fork,
     ) * 2**32 + domain_type
 ```
 
+#### `get_bitfield_bit`
+
+```python
+def get_bitfield_bit(bitfield: bytes, i: int) -> int:
+    """
+    Extract the bit in ``bitfield`` at position ``i``.
+    """
+    return (bitfield[i // 8] >> (7 - (i % 8))) % 2
+```
+
+#### `verify_bitfield`
+
+```python
+def verify_bitfield(bitfield: bytes, size: int) -> bool:
+    """
+    Verify ``bitfield`` against the ``size``.
+    """
+    if len(bitfield) != (size + 7) // 8:
+        return False
+
+    for i in range(size + 1, size - size % 8 + 8):
+        if get_bitfield_bit(bitfield, i) != 0:
+            return False
+
+    return True
+```
+
 #### `verify_slashable_vote_data`
 
 ```python
-def verify_slashable_vote_data(state: BeaconState, vote_data: SlashableVoteData) -> bool:
-    if len(vote_data.custody_bit_0_indices) + len(vote_data.custody_bit_1_indices) > MAX_CASPER_VOTES:
+def verify_slashable_vote_data(state: BeaconState, slashable_vote_data: SlashableVoteData) -> bool:
+    """
+    Verify validity of ``slashable_vote_data`` fields.
+    """
+    if slashable_vote_data.custody_bitfield != 0:  # [TO BE REMOVED IN PHASE 1]
```

Comment:
Is that the right way of checking the bitfield is all zeroes? @djrtwo

Reply:
Need to do check each byte in the bytes.

Can do:
```python
if not all(custody_byte == ZERO_HASH for custody_byte in slashable_vote_data.custody_bitfield):
```

`ZERO_HASH` might be poor usage because of the name `HASH` in it. Could use `b'\x00'` as well

Reply:
I would do `!= b'\x00' * len(slashable_vote_data.custody_bitfield)`